apps/finance/contracts/Finance.sol

```diff
@@ -105,23 +102,20 @@ contract Finance is AragonApp, ERC677Receiver {
      * @notice Allows to send ETH from this contract to Vault, to avoid locking them in contract forever.
      */
     function escapeHatch() public payable {
```

Comment:
We should make this function `function () payable { vault.deposit.value(this.balance)(ETH, msg.sender, this.balance, new bytes(0)) } `

Reply:
rather than escapeHatch

Reply:
So true! :100: